Mathlib/Algebra/Notation/README.md

```diff
@@ -0,0 +1,7 @@
+# Basic algebraic notation
+
+This subfolder contains definitions and very minimal theory related to the basic algebraic operators
+like `0`, `1`, `+`, `-`, `*`, `/`, `+ᵥ`, `•`, `-ᵥ`.
+
+The files in this subfolder should stay extremely in the import hierarchy, and in particular are
```

Comment:
Is there a word missing after "extremely"?

Reply:
I don't see you mean